wd_comp.c

```diff
@@ -495,8 +485,7 @@ int wd_do_comp_sync2(handle_t h_sess, struct wd_comp_req *req)
 			}
 			dbg("do, strm start, in =%u, out_len =%u\n",
 			    strm_req.src_len, strm_req.dst_len);
-			if (req->dst_len + strm_req.src_len > total_avail_out)
-				return -WD_ENOMEM;
```

Comment:
test

Reply:
This is just for check if dst space is enough for  hardware to write back. But this judgement  is not correctly maybe occur a false positive. Remove it just as this false positive?

Reply:
We just talked about this, the hardware can guarante that not write though memory., remove it is OK!